ingestion_pipelines/sirisx_situations_import_function/sirisx_situations_import_function/app.py

```diff
@@ -10,7 +10,7 @@
 from .shared.db import setup_db
 
 SIRI_SX_CANCELLATIONS_URL = (
-    "https://6tfu67dcng.execute-api.eu-west-2.amazonaws.com/v1/siri-sx"
+    "https://obg8kxwhq9.execute-api.eu-west-2.amazonaws.com/v1/siri-sx"
 )
```

Comment:
Update to:
```suggestion
siri_sx_cancellations_url = environ.get("SIRI_SX_CANCELLATIONS_URL")
```

Then update any usages of `SIRI_SX_CANCELLATIONS_URL` to `siri_sx_cancellations_url`